.travis.yml

```diff
@@ -68,6 +68,7 @@ script:
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "1" ]; then yarn build-storybook; fi
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "2" ]; then bin/test-console-check; fi
   - 'bin/knapsack_pro_rspec'
+  - '$(npm bin)/knapsack-pro-cypress'
```

Comment:
This is what will kickstart Cypress. I'm assuming since this runs after rspec, the web server is still running.